images/aquarium/config.xml

```diff
@@ -15,17 +15,19 @@
         <!-- Flavors -->
 	    <!-- <profile name="ContainerHost" description="Aquarium with container runtime (podman)"/> -->
         <!-- Platforms -->
-        <profile name="Vagrant-VirtualBox-x86_64" description="Vagrant for x86_64 VirtualBox" arch="x86_64"/>
-        <profile name="Vagrant-x86_64" description="Vagrant for x86_64 libvirt" arch="x86_64"/>
+        <profile name="Vagrant-VirtualBox-x86_64" description="Vagrant for
+        x86_64 with VirtualBox" arch="x86_64"/>
+        <profile name="Vagrant-x86_64" description="Vagrant for x86_64 with libvirt" arch="x86_64"/>
         <profile name="Vagrant" description="Vagrant" arch="x86_64">
             <requires profile="Vagrant-x86_64"/>
         </profile>
         <profile name="SelfInstall" description="Self Installing Image" arch="x86_64"/>
         <!-- Images (flavor + platform) -->
-        <profile name="Vagrant-VirtualBox" description="MicroOS with Podman for Vagrant" arch="x86_64">
+        <profile name="Vagrant-VirtualBox" description="Vagrant
+        with VirtualBox" arch="x86_64">
```

Comment:
See comment above.

Reply:
You are right, this is ugly. Didn't notice that with the editor for some reason :shrug: